### ISSUE TITLE
Require Contributing/Issue/Pull Request Documents

### ISSUE THREAD

**Issue by michaeltlombardi** [PowerShell/Phosphor#11]
The project should have useful templates for issues and pull requests, as well as a contribution guide to help people get running with contributing to the project.

**Comment by daviwil** [comment]
Yep, thanks!  I'll try to get this stuff together after the PowerShell Summit dust settles :)

**Comment by michaeltlombardi** [comment]
If you need help on the docs side, I have bandwidth!

**Comment by daviwil** [comment]
I would really appreciate your help there!  Hopefully in May we can really start cranking out some work on this and have something worth documenting :)

**Comment by michaeltlombardi** [comment]
I'm going to make sure I keep bandwidth available for documenting this project. I can steal from @devblackops in the meantime for the templates and submit a PR?

**Comment by daviwil** [comment]
Sure!  Sounds good man
